XamarinCommunityToolkit/Behaviors/RequiredStringValidationBehavior.shared.cs

```diff
@@ -0,0 +1,39 @@
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Behaviors
+{
+	public class RequiredStringValidationBehavior : ValidationBehavior
+	{
+		public static readonly BindableProperty ComparedToStringProperty =
+			BindableProperty.Create(nameof(ComparedToString), typeof(string),
+				typeof(RequiredStringValidationBehavior));
+
+		public static readonly BindableProperty ValidCommandProperty =
```

Comment:
Hm, I doubt this property is should be there.
Users can observe IsValid property changes directly instead of this command.